test/dynamo/test_misc.py

```diff
@@ -2114,14 +2114,14 @@ def forward(self, x, ref_id):
         correct_ref_id = id(m)
         opt_m = torch._dynamo.optimize(cnts, nopython=True)(m)
         opt_m(data, correct_ref_id)
-        self.assertEqual(cnts.op_count, 2)
+        self.assertEqual(cnts.op_count, 3)
```

Comment:
you can't do this, the test is run with both dynamic shapes enabled and not. Use ifdyn to toggle between the op counts